msg/CellularStatus.msg

```diff
@@ -1,28 +1,39 @@
-uint64 timestamp	# time since system start (microseconds)
+# Cellular status
+#
+# This is currently used only for logging a cell status
+# provided over MAVLink.
 
-uint8 CELLULAR_STATUS_FLAG_UNKNOWN=0 # State unknown or not reportable
-uint8 CELLULAR_STATUS_FLAG_FAILED=1 # velocity setpoint
-uint8 CELLULAR_STATUS_FLAG_INITIALIZING=2 # Modem is being initialized
-uint8 CELLULAR_STATUS_FLAG_LOCKED=3	# Modem is locked
-uint8 CELLULAR_STATUS_FLAG_DISABLED=4	# Modem is not enabled and is powered down
-uint8 CELLULAR_STATUS_FLAG_DISABLING=5	# Modem is currently transitioning to the CELLULAR_STATUS_FLAG_DISABLED state
-uint8 CELLULAR_STATUS_FLAG_ENABLING=6 	# Modem is currently transitioning to the CELLULAR_STATUS_FLAG_ENABLED state
-uint8 CELLULAR_STATUS_FLAG_ENABLED=7  # Modem is enabled and powered on but not registered with a network provider and not available for data connections
-uint8 CELLULAR_STATUS_FLAG_SEARCHING=8  # Modem is searching for a network provider to register
-uint8 CELLULAR_STATUS_FLAG_REGISTERED=9  # Modem is registered with a network provider, and data connections and messaging may be available for use
-uint8 CELLULAR_STATUS_FLAG_DISCONNECTING=10  # Modem is disconnecting and deactivating the last active packet data bearer. This state will not be entered if more than one packet data bearer is active and one of the active bearers is deactivated
-uint8 CELLULAR_STATUS_FLAG_CONNECTING=11  # Modem is activating and connecting the first packet data bearer. Subsequent bearer activations when another bearer is already active do not cause this state to be entered
-uint8 CELLULAR_STATUS_FLAG_CONNECTED=12  # One or more packet data bearers is active and connected
+uint64 timestamp # Time since system start [us]
 
-uint8 CELLULAR_NETWORK_FAILED_REASON_NONE=0 # No error
-uint8 CELLULAR_NETWORK_FAILED_REASON_UNKNOWN=1 # Error state is unknown
-uint8 CELLULAR_NETWORK_FAILED_REASON_SIM_MISSING=2 # SIM is required for the modem but missing
-uint8 CELLULAR_NETWORK_FAILED_REASON_SIM_ERROR=3 # SIM is available, but not usable for connection
+uint16 status # Status bitmap 1: Roaming is active [@enum STATUS_FLAG]
```

Comment:
What does "Status bitmap 1: Roaming is active" mean? - i.e. why "1" and what does "roaming is active" mean?

Generally I would expect something like this, where the enum values themselves define what is meaningful.

```suggestion
uint16 status # Status bitmap [@enum STATUS_FLAG]
```

Reply:
You're totally right, I have no idea why it would say "Roaming is active" there. I assume it's some copy pasta error that I had copied again without questioning enough. Generally the message seems to only be used to log the content of https://mavlink.io/en/messages/common.html#CELLULAR_STATUS in the PX4 ulog (which I don't think is such a good reason for it to use up everyone's flash but also in the MAVLink message there's no word about roaming.